app/assets/stylesheets/form.css

```diff
@@ -0,0 +1,47 @@
+
+.review-form{
+  margin-top: 20rem;
+  font-style: italic;
+  color: red;
+  font-size: 1rem;
+}
+
+.submit-button{
+  color: black;
+}
+
+.forgot-pass-form{
+  margin-top: 3rem;
+  margin-left: 2rem;
+  font-style: italic;
+  color: #ffffff;
+  font-size: 4rem;
+}
+.field {
+  color: white;
+  font-weight: 800;
+}
+.sign_up {
```

Comment:
names should have dashes not hyphens, see style guide here: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters